pkgs/top-level/all-packages.nix

```diff
@@ -20980,4 +20980,6 @@ with pkgs;
   simplehttp2server = callPackage ../servers/simplehttp2server { };
 
   diceware = callPackage ../tools/security/diceware { };
+
+  xml2rfc = pythonPackages.callPackage ../tools/typesetting/xml2rfc { };
```

Comment:
either use `pythonPackages.callPackage` and make sure there are parameters for all dependencies, or use `callPackage` and pass in only `python` and `stdenv`.

Reply:
I typically recommend the latter in case of applications, because it may happen you need to override a version of a dependency in the expression

Reply:
What about aws_shell, ddar, wakatime, pyprof2calltree, udocker, almonds, sc-controller? The seem to use `pytjonPackages.callPackage` without parameters for dependencies.

Reply:
Nevermind :) I will re-work my patch.

Reply:
No, it's just that sometimes mistakes slip in.